### ISSUE TITLE
Can't create sublevel lists item without html

### ISSUE THREAD

**Issue by gabrielzschmitz** [blogc/blogc#15]
Don't know if I'm dumb or what, but I can't create a lists with sublevels without using html. A demo here:

Using markdown:
file.txt:
```
- First item
- Second item
- Third item
    - Indented item
    - Indented item
- Fourth item
```

output:
First item
Second item
Third item - Indented item - Indented item
Fourth item

Using HTML:
file.txt:
```
<ul>
  <li>First item</li>
  <li>Second item</li>
  <li>Third item
    <ul>
      <li>Indented item</li>
      <li>Indented item</li>
    </ul>
  </li>
  <li>Fourth item</li>
</ul>
```

output:
- First item
- Second item
- Third item
    - Indented item
    - Indented item
- Fourth item

**Comment by Joursoir** [comment]
[blogc-source](https://blogc.rgm.io/man/blogc-source.7.html) says nothing about it, so I guess it's not supported. But this feature is really needed

**Comment by rafaelmartins** [comment]
yes, this is not supported, because lists are block elements, and the list items are parsed as inline elements. allowing block elements inside list items is not really a good idea, then I need to handle the sub lists as a special case. I'll look for a solution
